### ISSUE TITLE
Multi-color bitmap field

### ISSUE THREAD

**Issue by romw314** [RaspberryPiFoundation/blockly-samples#2473]
### Check for duplicates

- [x] I have searched for similar issues before opening a new one.

### Component

`@blockly/field-bitmap`

### Request

The bitmap field could support multiple colors. That would be super useful for some projects.

### Alternatives considered

Creating my own field that extends the bitmap field. But it requires a lot of work.

**Comment by cpcallen** [comment]
Yeah, that would be a nice feature!  Unfortunately we probably won't have time to implement it ourselves, but you should take a look at MakeCode (which we believe has something similar).  We'd be happy to accept well-written PRs adding this feature.
